app/bundles/LeadBundle/Form/Type/CampaignEventPointType.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Mautic\LeadBundle\Form\Type;
+
+use Mautic\LeadBundle\Provider\TypeOperatorProviderInterface;
+use Mautic\LeadBundle\Segment\OperatorOptions;
+use Mautic\PointBundle\Form\Type\GroupListType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @extends AbstractType<CampaignEventPointType>
+ */
+class CampaignEventPointType extends AbstractType
+{
+    private TypeOperatorProviderInterface $typeOperatorProvider;
+
+    public function __construct(TypeOperatorProviderInterface $typeOperatorProvider)
+    {
+        $this->typeOperatorProvider = $typeOperatorProvider;
+    }
```

Comment:
Feel free to use shortened controllers in M5
```suggestion
    public function __construct(private TypeOperatorProviderInterface $typeOperatorProvider)
    {
    }
```